composer.json

```diff
@@ -28,29 +28,29 @@
     "source": "https://github.com/contao-community-alliance/translator"
   },
   "require": {
-    "php": ">=5.3",
-    "contao/core": ">=3.2,<3.6-dev",
-    "contao-community-alliance/composer-plugin": "~2.0",
-    "contao-community-alliance/event-dispatcher": "~1.0",
-    "contao-community-alliance/events-contao-bindings": "~3.0"
+    "php": ">=5.6",
+    "contao/core-bundle": "~4.4",
+    "contao-community-alliance/event-dispatcher": "~2.0@dev"
   },
   "require-dev": {
-    "phpcq/all-tasks": "~1.1"
+    "phpcq/all-tasks": "~1.1",
+    "contao/manager-bundle": "~4.4"
   },
   "autoload": {
-    "psr-0": {
-      "ContaoCommunityAlliance\\Translator": "src"
+    "psr-4": {
+      "ContaoCommunityAlliance\\Translator\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "ContaoCommunityAlliance\\Translator\\Test\\": "tests/"
     }
   },
   "extra": {
     "branch-alias": {
-      "dev-master": "2.0.x-dev"
+      "dev-feature/contao4": "2.1.x-dev"
```

Comment:
I didn't get it before that increasing the next minor should be the way to go. Because we drop support of Contao 3.5 I personally prefer the next major. If we support C3.5 and C4 I would use the next minor.

People already have to adjust ther dependencies if they want to support C4. So it shouldn't be too much work adding anonther version constraint.

Well but I can live with release 2.1 as well.